pkg/bsp-imx/patches/uboot-patches-lf_v2022.04/0015-arch-arm-dts-Add-SMBIOS-information-for-Advantech-s-.patch

```diff
@@ -0,0 +1,61 @@
+From f87eab4b21306b75e003902c6dcb106a45b90b4f Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Ren=C3=AA=20de=20Souza=20Pinto?= <rene@renesp.com.br>
+Date: Wed, 11 Jan 2023 15:09:35 +0100
+Subject: [PATCH 15/17] arch: arm: dts: Add SMBIOS information for Advantech's
+ board
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+- Add device tree node for Advantechs's EPC3720 device with SMBIOS
+  information.
+- Enable SMBIOS in imx8mp_rsb3720a1_6G_defconfig
```

Comment:
@eriknordmark there is nothing wrong with these lines, they are part of the commit message, GitHub's diff just get a bit confused because of the -, so it shows it in red, but there is no problem for merge, etc.